src/integrationTest/resources/application-integration.yaml

```diff
@@ -0,0 +1,47 @@
+spring:
+  cloud:
+    azure:
```

Comment:
I wonder if we need this azure part as seems similar to the main app yaml. If not already could you try removing and see if this still works?

Reply:
Removed